packages/components/src/logos/jetpack-logo/index.tsx

```diff
@@ -57,13 +64,13 @@ const LogoPathSize32Monochrome: React.FunctionComponent< any > = () => (
 	</>
 );
 
-const JetpackLogo: React.FunctionComponent< JetpackLogoProps > = ( {
+export const JetpackLogo = ( {
```

Comment:
This needs to be a named export for Storybook to extract TS type data for auto-generated Controls.